components/atoms/Accordion.js

```diff
@@ -36,10 +52,10 @@ export function Accordion(props) {
           </div>
           <div className="ml-8">
             <h2
-              id={props.id}
               className={`ml-6 (${props.titleClass}) ? ${props.titleClass} : ''`}
             >
-              {props.title}
+              {props.title}, open = {openFlag ? "true" : "false"} arrow ={" "}
```

Comment:
I know this is probably just for testing purposes but we should get rid of the open = state, arrow = state once this is working.